### DISCUSSION TITLE
Use with other pretrained models

### DISCUSSION THREAD

**Discussion by stobermeier** [nerdyrodent/VQGAN-CLIP#104] [Ideas]
Is it possible to run the tool with other pretrained models? [This repository](https://github.com/CompVis/taming-transformers) provides a list of several other pretrained models. However, when trying to use the tool with CelebA-HQ, I run into the error message "ModuleNotFoundError: No module named 'transformers'".

Thank you for your great work!

**Comment by jonnm4** [comment]
I get a rainbow square that never changes; iteration to iteration staying the same.
![output](https://user-images.githubusercontent.com/20403377/187043815-186d391e-84a1-4ed0-8ef8-39433e2f52b7.png)
